src/Stripe.net/Services/Charges/StripeChargeLevel3LineItemOptions.cs

```diff
@@ -0,0 +1,25 @@
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeChargeLevel3LineItemOptions : INestedOptions
```

Comment:
I still don't understand when something should be called Options versus Option.

Reply:
I think it should always be `Options`.

Reply:
Yeah that would work. I thought that we started using `Option` for nested ones to differentiate them from the top-level ones.